server/neptune/temporalworker/config/config.go

```diff
@@ -20,9 +20,10 @@ type ServerConfig struct {
 }
 
 type TerraformConfig struct {
-	DefaultVersionStr      string
-	DefaultVersionFlagName string
-	DownloadURL            string
+	TFDefaultVersion       string
+	TFDownloadURL          string
+	ConftestDefaultVersion string
+	ConftestDownloadURL    string
```

Comment:
I don't think this is a good place for this.  This is intentionally called terraform config lol

Can you move this to like ValidationConfig which can contain all the policy stuff from the global config and this stuff as well.

Reply:
Yeah so this looked to be the configurations needed for the "Terraform Workflow" not "Terraform activities", similar to how there is a "DeploymentConfig" struct for deploy workflows. But that's reasonable, will try to move.